src/main/java/io/mosip/mimoto/controller/CredentialsController.java

```diff
@@ -79,4 +82,33 @@ public ResponseEntity<?> downloadCredentialAsPDF(
         }
     }
 
+    @PostMapping("/verify")
+    public ResponseEntity<?>verifyCredential(@RequestBody String credential){
```

Comment:
can we use VCCredentialsResponseDTO for requestBody type instead of string?

Reply:
Done